gh_pages/_source/session8/Motion-Planning-with-Tesseract.rst

```diff
@@ -0,0 +1,435 @@
+   
+   Within ``snp_motion_planning/src/planner_profiles.hpp``, find the section
+
+   .. code-block:: c++
+
+      /* ========================================
+       * Fill Code: ADD CUSTOM PLANNER PROFILES
+       * ========================================*/
+
+   Copy and past the following below:
+
+   .. code-block:: c++
+
+   	profile_dict_->addProfile<tesseract_planning::DescartesPlanProfile<float>>(
+          tesseract_planning::profile_ns::DESCARTES_DEFAULT_NAMESPACE, PROFILE, createDescartesPlanProfile<float>());
+
+   This line adds your new custom planning profile to the planning server for our motion plan.
+
+#. Add the planner to the taskflow:
+   
+   Navigate to ``snp_motion_planning/taskflow_generators.hpp`` and find the method ``ctor()`` within the class ``CartesianMotionPipelineTask``. Find the following block inside
+
+   .. code-block:: c++
+
+      /* ========================================
+       * Fill Code: CREATE CUSTOM PLANNER NODES
+       * ========================================*/
+
+   Below the block add the following
+
+   .. code-block:: c++
+
+      boost::uuids::uuid descartes_planner_task =
+            addNode(std::make_unique<tesseract_planning::DescartesMotionPlannerTask>(output_keys_[0], output_keys_[0],
+            false));
+
+   Now we have created nodes for our planner. Find 
+
+   .. code-block:: c++
+
+      /* =======================
+       * Fill Code: EDIT EDGES
+       * =======================*/
+
+   in the same method and fill in the code to add edges between our nodes. Will need to replace 
+
+   .. code-block:: c++
+
+      addEdges(min_length_task, { contact_check_task });
+
+   with
+
+   .. code-block:: c++
+
+      addEdges(min_length_task, { descartes_planner_task })
+
+   and then add the line
+
+   .. code-block:: c++
+
+      addEdges(descartes_planner_task, { error_task, contact_check_task });
+
+   Because we want Descartes to be run globally over the entire motion, we need to create a new global raster pipeline. Toward the bottom of the file find
+
+   .. code-block:: c++
+
+      /* =========================================
+       * Fill Code: CUSTOM GLOBAL RASTER PIPELINE
+       * =========================================*/
+
+   Under it add the following line
+
+   .. code-block:: c++
+
+      using CustomGlobalRasterGlobalPipeline =
+          tesseract_planning::RasterGlobalPipelineTask<tesseract_planning::SimpleMotionPipelineTask,
+                                                       tesseract_planning::DescartesGlobalMotionPipelineTask,
+                                                       CustomRasterPipeline>;
+
+   We have now added Descartes to our raster taskflow. 
+
+   .. Note:: Pay attention to how the graph's edges and vertices are connected. We have already included post-collision checking for the simple planner and time parameterization. Play around with removing one or both of those and observe how your motion plan changes. 
```

Comment:
@marrts recommends that we have this included by default and dont have this as a fill code in order to ensure snp_motion_planning can build by default @tiffanyec